cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```
@@ -256,6 +262,26 @@ objects:
       usePodIdentity: "false"
     provider: azure
 
+- apiVersion: secrets-store.csi.x-k8s.io/v1
+  kind: SecretProviderClass
+  metadata:
+    name: arm-identity
+    namespace: ${NAMESPACE}
+  spec:
+    parameters:
+      clientID: ${ARM_HELPER_IDENTITY_CLIENT_ID}
```

```suggestion
      clientID: ${AZURE_CS_MI_CLIENT_ID} 
```
The CSI driver requires the Azure identity, the ARM identity is used in CS.

---

fixed